Source/JavaScriptCore/API/JSValue.h

```diff
@@ -169,23 +169,61 @@ NS_CLASS_AVAILABLE(10_9, 7_0)
 @param context The JSContext to which the resulting JSValue belongs.
 @result The JSValue representing a unique JavaScript value with type symbol.
 */
-+ (JSValue *)valueWithNewSymbolFromDescription:(NSString *)description inContext:(JSContext *)context JSC_API_AVAILABLE(macos(10.15), ios(13.0));
++ (JSValue * _Null_unspecified)valueWithNewSymbolFromDescription:(NSString * _Null_unspecified)description inContext:(JSContext * _Null_unspecified)context JSC_API_AVAILABLE(macos(10.15), ios(13.0));
+
+/*!
+@method
+@abstract Create a new, unique, bigint object.
+@param string The string representation of the bigint object being created.
+@param context The JSContext to which the resulting JSValue belongs.
+@result The JSValue representing a unique JavaScript value with type bigint.
+@discussion This is equivalent to calling the `BigInt` constructor from JavaScript with a string argument.
+*/
++ (JSValue * _Nullable)valueWithNewBigIntFromString:(NSString * _Nonnull)string inContext:(JSContext * _Nonnull)context JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
+
+/*!
+@method
+@abstract Create a new, unique, bigint object.
+@param int64 The signed 64-bit integer of the bigint object being created.
+@param context The JSContext to which the resulting JSValue belongs.
+@result The JSValue representing a unique JavaScript value with type bigint.
+*/
++ (JSValue * _Nullable)valueWithNewBigIntFromInt64:(int64_t)int64 inContext:(JSContext * _Nonnull)context JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
+
+/*!
+@method
+@abstract Create a new, unique, bigint object.
+@param uint64 The unsigned 64-bit integer of the bigint object being created.
+@param context The JSContext to which the resulting JSValue belongs.
+@result The JSValue representing a unique JavaScript value with type bigint.
+*/
++ (JSValue * _Nullable)valueWithNewBigIntFromUInt64:(uint64_t)uint64 inContext:(JSContext * _Nonnull)context JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
```

Comment:
Ditto for uint64_t.

Reply:
It seems that Obj-C has uint64_t and int64_t.
https://developer.apple.com/documentation/kernel/uint64_t/
https://developer.apple.com/documentation/kernel/int64_t/

Reply:
For swift, we have `int64` and `uint64`.
https://developer.apple.com/documentation/swift/int64/
https://developer.apple.com/documentation/swift/uint64/